docs/docs/tutorials/simple_hello_world.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: Creating a simple hello world with gRPC-Gateway
+parent: Tutorials
+nav_order: 4
+---
+
+### Creating a simple hello world with gRPC-Gateway
+
+### gRPC-Gateway
+
+We all know that gRPC is not a tool for everything. There are cases where we still want to provide a traditional RESTful JSON API. The reasons can range from maintaining backwards-compatibility to supporting programming languages or clients not well supported by gRPC. But coding another API for REST is quite a time consuming and tedious.
+
+So is there any way to code just once, but can provide APIs in both gRPC and REST at the same time?
+
+The answer is Yes.
+
+### Usage
+
+One way to achieve that is to use gRPC gateway. gRPC gateway is a plugin of the protocol buffer compiler. It reads the protobuf service definitions and generates a proxy server, which translates a RESTful HTTP call into gRPC request.
+
+All we need to do is a small amount of configuration
+in the service. Before start coding, we have to install some
+tools.
+
+Use `go get -u` to download the following packages:
+
+```sh
+go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
+go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
+go get -u github.com/golang/protobuf/protoc-gen-go
+```
+
```

Comment:
```suggestion

This installs the protoc generator plugins we need to generate the stubs. Make sure to add `$GOPATH/bin` to your `$PATH` so that executables installed via `go get` are available on your `$PATH`.
```

Reply:
Everything above this line should on a separate page about "pre-requirements".